strawberry/http/async_base_view.py

```diff
@@ -112,6 +112,14 @@ async def execute_operation(
 
         assert self.schema
 
+        return await self.schema.subscribe(
```

Comment:
here we need to check if we are doing a subscription or not

Reply:
also I've noticed that if I try to do a subscription using schema.execute it doesn't error, but it does something weird, worth adding a check for this case.